.github/workflows/release.yml

```diff
@@ -1,36 +1,39 @@
 name: Release
 
-on:
-  push: { tags: ["v*"] }
+on: { push }
```

Comment:
Do we definitely want to run this for every push rather than just for tags?

Reply:
Note line 33.